File: hw/chavez/u1/hw03ChickenFarmSimulation/javascript/view/ChickenFarmSimulation.js
const { Chicken } = require("../model/Chicken.js");

class ChickenFarmSimulation {
    mostrar() {
        this.chicken = new Chicken(1, "Lucy", "White and brown", 1, true);
        console.log("My Chicken is ---> " + this.chicken.toString());
    }
}

module.exports = { ChickenFarmSimulation };

File: hw/chavez/u1/hw02ChickenFarmSimulation/python/view/ChickenFarmSimulation.py
import sys
import os
sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from model.Chicken import Chicken
def main():
    chicken = Chicken(1, "Lucy", "White and brown", 1, True)
    print("My chicken is -->" + chicken.toString())
if __name__ == "__main__":    main()